### ISSUE TITLE
log info content is missing

### ISSUE THREAD

**Issue by xiaotupansy** [jptoto/log4net.ElasticSearch#65]
Hi,

I found an issue that I can't find log info content in es.

Here's the log code:
```
public ActionResult Index()
        {
            _log.Info("test!", new ApplicationException("shit went down and it wasn't good")); 
            return View();
        }
```

And here's record in es,you can find that the message is null!

{

    "_index": "testlog",
    "_type": "logEvent",
    "_id": "AVSj2CU-lQLA0uGOM-km",
    "_version": 1,
    "_score": 1,
    "_source": {
        "timeStamp": "2016-05-12T07:22:12.1540257Z",
        "message": null,
        "messageObject": { },
        "exception": {
            "Type": "System.ApplicationException",
            "Message": "shit went down and it wasn't good",
            "HelpLink": null,
            "Source": null,
            "HResult": -2146232832,
            "StackTrace": null,
            "Data": { },
            "InnerException": null
        },
        "loggerName": "ESLogger",
        "domain": null,
        "identity": "",
        "level": "INFO",
        "className": "Log4netWithElasticSearch.Controllers.HomeController",
        "fileName": "c:\Users\tu\Desktop\TuStudy\Log4netWithElasticSearch\Controllers\HomeController.cs",
        "lineNumber": "16",
        "fullInfo": "Log4netWithElasticSearch.Controllers.HomeController.Index(c:\Users\tu\Desktop\TuStudy\Log4netWithElasticSearch\Controllers\HomeController.cs:16)",
        "methodName": "Index",
        "fix": "None",
        "properties": {
            "log4net:Identity": "",
            "log4net:UserName": "tu-PC\tu",
            "log4net:HostName": "tu-PC",
            "@timestamp": "2016-05-12T07:22:12.1540257Z"
        },
        "userName": "tu-PC\tu",
        "threadName": "9",
        "hostName": "TU-PC"
    }

}

**Comment by jptoto** [comment]
@xiaotupansy I agree the message should also be included in the `_source.message` field even if the message is rendered as an `Exception()` object. I'll work on this.

**Comment by jptoto** [comment]
@xiaotupansy I've tried this on both a console project and asp.net MVC and can't seem to duplicate the results. The `RenderedMessage` should be appearing in the `_source.message` field for you. Can you tell me which version of log4net, MVC, and Elasticsearch your're using?

**Comment by xiaotupansy** [comment]
@jptoto Here's the config for mvc project:

<package id="log4net" version="2.0.5" targetFramework="net45" />
  <package id="log4net.ElasticSearch" version="2.3.3" targetFramework="net45" />
  <package id="Microsoft.AspNet.Mvc" version="5.2.2" targetFramework="net45" />

And the version of Elasticsearch is 2.3.2.

**Comment by xiaotupansy** [comment]
And here's the configuration:

<log4net>
    <appender name="ElasticSearchAppender" type="log4net.ElasticSearch.ElasticSearchAppender, log4net.ElasticSearch">
      <connectionString value="Server=localhost;Index=testlog;Port=9200;"/>
      <Fix value="0"/>
      <bufferSize value="3" />
    </appender>
    <appender name="textLogger" type="log4net.Appender.RollingFileAppender">
      <file value="C:/temp/testlog.log" />
      <rollingStyle value="Date" />
      <datePattern value=".yyyyMMdd" />
      <layout type="log4net.Layout.PatternLayout">
        <conversionPattern value="%date [%thread] %-5level %logger property:[%property{NDC}] -%message %location%newline" />
      </layout>
    </appender>
    <logger name="ESLogger">
      <level value="INFO" />
      <appender-ref ref="textLogger" />
      <appender-ref ref="ElasticSearchAppender" />
    </logger>
  </log4net>
